Rules.md

```diff
@@ -143,22 +142,15 @@ Token types:
 
 ## Moving units
 
-Units can move through unoccupied, friendly and sea regions. Moving to an enemy region consumes all
-Move Points and initiates a battle.
+Units can move to any adjacent region, except for friendly regions owned by other players.
 
-### Move Points (MP)
+### Roads
 
-Moves between regions normally cost 2 MP.<br>
-Moves starting in a friendly region with at least one coin invested cost 1 MP.<br>
-The total amount of MP consumed may never exceed the given amount (typically 2).
+Moves can hop over a single friendly region if a road exists between the starting region and that
+friendly region.<br>
```

Comment:
Pedantic nit: Remove < br> and replace it by a newline or put it in the same paragraph.